.drone.yml

```diff
@@ -92,6 +92,40 @@ trigger:
     - pull_request
     - push
 
+---
+kind: pipeline
+name: qt-5.15-clazy
+
+steps:
+- name: cmake
+  image: ghcr.io/nextcloud/continuous-integration-client:client-5.15-2
+  volumes:
+    - name: build
+      path: /drone/build
+  commands:
+    - cd /drone/build
+    - cmake -GNinja -DCMAKE_C_COMPILER=clang-10 -DCMAKE_CXX_COMPILER=clazy -DCMAKE_BUILD_TYPE=Debug -DBUILD_UPDATER=ON -DBUILD_TESTING=1 -DECM_ENABLE_SANITIZERS=address -DCMAKE_CXX_FLAGS=-Werror ../src
+- name: compile
+  image: ghcr.io/nextcloud/continuous-integration-client:client-5.15-2
+  volumes:
+    - name: build
+      path: /drone/build
+  commands:
+    - cd /drone/build
+    - CLAZY_IGNORE_DIRS=".*3rdparty.*|.*build.*" ninja
```

Comment:
better use this [clazy-standalone](https://github.com/KDE/clazy#clazy-standalone-and-json-database-support)
that should be faster than doing a real compilation (that should not be needed we already compile with clang before